ios/engine/KMEI/KeymanEngine/Classes/Resource Management/AssociatingPackageInstaller.swift

```diff
@@ -208,7 +208,21 @@ public class AssociatingPackageInstaller<Resource: LanguageResource, Package: Ty
                 withAssociators associators: [Associator] = [],
                 progressReceiver: @escaping ProgressReceiver) {
     self.package = package
-    self.defaultLgCode = defaultLanguageCode
+    if let defaultLanguageCode = defaultLanguageCode {
+      if package.languages.contains(where: { $0.id == defaultLanguageCode }) {
+        self.defaultLgCode = defaultLanguageCode
+      } else if let match = package.languages.first(where: { $0.id.contains(defaultLanguageCode) }) {
```

Comment:
Does this mean that BCP 47 tag `sen` would match `en`? That doesn't seem right...

Reply:
Good point... I should have used `.hasPrefix` instead of `.contains`.